src/state/disconnect-toast-mixin.ts

```diff
@@ -61,11 +69,81 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
 
     protected hassDisconnected() {
       super.hassDisconnected();
-
       showToast(this, {
         message: this.hass!.localize("ui.notification_toast.connection_lost"),
         duration: 0,
         dismissable: false,
       });
     }
+
+    private _handleMessage(message: any): void {
+      if (this._dismissed || this.hass!.config.state !== STATE_NOT_RUNNING) {
```

Comment:
Let's instead unsubscribe when you dismiss

Reply:
done